static/app/views/dashboardsV2/widgetBuilder/buildSteps/dataSetStep.tsx

```diff
@@ -11,7 +12,7 @@ import {BuildStep} from './buildStep';
 
 const DATASET_CHOICES: [DataSet, string][] = [
   [DataSet.EVENTS, t('Events (Errors, transactions)')],
```

Comment:
Part of the copy updates was changing this data set name. Is this reflected anywhere in the PR?

Reply:
Ah, I remembered seeing it but I noticed in the Notion doc it was removed by you (probably an accident) and was going to follow up. I'll update it in this PR, "Errors and Transactions" is what we agreed on, correct?

Reply:
https://github.com/getsentry/sentry/pull/33530/files#diff-7bbe835240512c55ba82968c38e19c27d0366c86831d3304874bf80bb57612e6R14

The change is up now :+1:

Reply:
Def an accident. Thanks Nar!